### PR TITLE
Update create-integration.md

### PR DESCRIPTION
**Pull request by hellotaran**
## Purpose of this pull request

This pull request (PR) is to provide clarifications for creating an integration #5611

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/get-started/create-integration.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by keharper** [comment]
@hellotaran The linter is failing on lines 168 and 170. These lines are registering as 4 blank spaces instead of blank lines. You'll need to correct those.

Also, please make the same updates in the v2.2 version of the file.

File: guides/v2.2/get-started/create-integration.md [hunk 1]
```diff
@@ -172,6 +172,18 @@ To develop a module, you must:
 
     `testIntegration` must refer to your `etc/integration.xml` file, and the integration name value must be the same.
 
+    The following example demonstrates a minimal config.xml file.
```
Comment (keharper) [inline guides/v2.2/get-started/create-integration.md]:
```suggestion
    The following example demonstrates a minimal `config.xml` file.
```

File: guides/v2.3/get-started/create-integration.md [hunk 2]
```diff
@@ -164,7 +164,19 @@ To develop a module, you must:
 
     `$this->integrationManager->processIntegrationConfig(['testIntegration']);`
 
-    `testIntegration` must refer to your `etc/integrations/config.xml` file, and the integration name value must be the same.
+    `testIntegration` must refer to your `etc/integration/config.xml` file, and the integration name value must be the same.
+
+    The following example demonstrates a minimal config.xml file.
```
Comment (keharper) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
    The following example demonstrates a minimal `config.xml` file.
```

File: guides/v2.2/get-started/create-integration.md [hunk 3]
```diff
@@ -172,6 +172,18 @@ To develop a module, you must:
...
+
+    ```xml
+    <integrations>
+      <integration name="TestIntegration">
+         <email>someone@example.com</email>
+         <endpoint_url>http://example.com</endpoint_url>
```
Comment (keharper) [inline guides/v2.2/get-started/create-integration.md]:
```suggestion
         <endpoint_url>https://example.com</endpoint_url>
```

File: guides/v2.2/get-started/create-integration.md [hunk 4]
```diff
@@ -172,6 +172,18 @@ To develop a module, you must:
...
+         <identity_link_url>http://example.com/identity_link_url</identity_link_url>
```
Comment (keharper) [inline guides/v2.2/get-started/create-integration.md]:
```suggestion
         <identity_link_url>https://example.com/identity_link_url</identity_link_url>
```

File: guides/v2.3/get-started/create-integration.md [hunk 5]
```diff
@@ -164,7 +164,19 @@ To develop a module, you must:
...
+
+    ```xml
+   <integrations>
+      <integration name="TestIntegration">
+         <email>someone@example.com</email>
+         <endpoint_url>http://example.com</endpoint_url>
```
Comment (keharper) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
         <endpoint_url>https://example.com</endpoint_url>
```

File: guides/v2.3/get-started/create-integration.md [hunk 6]
```diff
@@ -164,7 +164,19 @@ To develop a module, you must:
...
+         <identity_link_url>http://example.com/identity_link_url</identity_link_url>
```
Comment (keharper) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
         <identity_link_url>https://example.com/identity_link_url</identity_link_url>
```

Comment (hellotaran) [reply guides/v2.3/get-started/create-integration.md]: Done

File: guides/v2.3/get-started/create-integration.md (hunk 5 above)
Comment (hellotaran) [reply guides/v2.3/get-started/create-integration.md]: Done

File: guides/v2.2/get-started/create-integration.md (hunk 4 above)
Comment (hellotaran) [reply guides/v2.2/get-started/create-integration.md]: Done

File: guides/v2.2/get-started/create-integration.md (hunk 3 above)
Comment (hellotaran) [reply guides/v2.2/get-started/create-integration.md]: Done

File: guides/v2.3/get-started/create-integration.md (hunk 2 above)
Comment (hellotaran) [reply guides/v2.3/get-started/create-integration.md]: Done

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @hellotaran

**Pull request merged**